Plugins/ExaTrkX/src/ExaTrkXPipeline.cpp

```diff
@@ -28,8 +28,9 @@ ExaTrkXPipeline::ExaTrkXPipeline(
     throw std::invalid_argument("Missing track building module");
   }
   if (m_edgeClassifiers.empty() or
-      not std::all_of(m_edgeClassifiers.begin(), m_edgeClassifiers.end(),
-                      [](const auto &a) { return static_cast<bool>(a); })) {
+      not std::ranges::all_of(m_edgeClassifiers, [](const auto &a) {
+        return static_cast<bool>(a);
+      })) {
```

Comment:
Can we replace "not all of" by "any not of", as this allows a quicker short-circuit?